### DISCUSSION TITLE
click & open tracking statistics?

### DISCUSSION THREAD

**Discussion by tchwpkgorg** [postalserver/postal#1514] [Feature suggestions]
I've enabled click & open tracking, it works, but I'm not sure how to view the results.

I can see that on the "Messages" page, Postal adds "Opened" tag for an email which was tracked.

However, is there a page which displays some statistics about it (i.e. per day, per domain etc.)?
If not - perhaps it's possible to look it up in a database somehow?

**Comment by willpower232** [comment]
As you mentioned the UI only exposes the tags on the messages to show you what happened to the messages from an administrative view.

Unfortunately there are no statistics at this time but there has been a lot of requests over numerous issues so hopefully it can get addressed at some point.

The best way of compiling this information is to make use of the webhooks functionality and post events to some code and make your own logs. I am not sure if Postals own database will currently be able to give you the same depth of information.

**Comment by tchwpkgorg** [comment]
For the record, this seems to sit in these two tables:

```
select * from clicks\G
```

```
select * from loads\G
```

**Comment by tchwpkgorg** [comment]
By the way - it it at least possible to find clicked / opened messages in the web interface?

I'm thinking of something like "status:opened", similar to "status:sent"

**Comment by willpower232** [comment]
That doesn't appear to be possible at the minute unfortunately.
